backend/controllers/Event.js

```diff
@@ -36,7 +37,7 @@ const createEvent = async (req, res) => {
       //Make sure all promise is fullfilled
       await Promise.all(invitedParticipation);
     }
-    res.status(201).json({ event: event });
+    res.status(201).json(event.toJSON());
```

Comment:
are you sure this is needed?
`json({ event: event })` will look like
```
data: {
  event: {
    name: ...,
    description: ...,
    etc
  }
}
```
And I might be wrong but `json(event)` will be enough for this
```
data: {
  name: ...,
  description: ...,
  etc
}
```

Reply:
Oh my bad, appreciate your feedback!